lib/models/home/shortcuts/shortcuts_model.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'shortcuts_model.freezed.dart';
+part 'shortcuts_model.g.dart';
+
+@unfreezed
```

Comment:
It is because while reordering the item in grid view, it threw me an error.

Reply:
Ah, ok that makes sense, but I think it exposes a weakness in our architecture. We're currently using the same model for both backend data retrieval and UI state management. This approach could lead to some challenges:

**Immutable vs Mutable:** Our backend model, ideally, should be immutable to maintain a single source of truth. Making it mutable for UI purposes can introduce unpredictable state changes and complexity.

**Separation of Concerns:** By using the same model for both backend and UI, we're blurring the lines between our data layer and presentation layer. This goes against the separation of concerns principle, potentially making our codebase harder to maintain.

**State Management:** With our current setup, any changes in the UI directly affect our model state. In a reactive framework like Flutter, coupled with Riverpod, this might lead to unintended side effects and hard-to-track bugs.

I’ve also noticed that we have a similar implementation elsewhere in the codebase (TrackModel?). My suggestion would be to differentiate between the backend model and the UI model. We could keep our backend model (@freezed and immutable) to represent the data as it is in the backend. For the UI, we can introduce a separate, mutable model that reflects the user interactions and UI state. This approach would not only improve the maintainability and predictability of our app but also aligns better with best practices in app architecture.

I'll create a ticket for us to review and update the other instances where we've used @unfreezed, and we can do it in another PR.

wdyt?